test/csuite/wt6185_modify_ts/main.c

```diff
@@ -116,6 +119,32 @@ mmrand(u_int min, u_int max)
     return (v);
 }
 
+/*
+ * change_key --
+ *     Switch to a different key.
+ */
+static void
+change_key(u_int n)
+{
+    if (use_columns)
+        keyrecno = n + 1;
+    else
+        testutil_check(__wt_snprintf(keystr, sizeof(keystr), "%010u.key", n));
+}
+
+/*
+ * set_key --
+ *     Set the current key in the cursor.
+ */
+static void
+set_key(WT_CURSOR *c)
+{
+    if (use_columns)
+        c->set_key(c, keyrecno);
+    else
+        c->set_key(c, keystr);
+}
```

Comment:
We could actually detect if the cursor points to a column-store, rather than using the global variable, if that makes anything easier.

Reply:
It doesn't :-)